app/src/main/java/com/android/unio/ui/association/AssociationProfile.kt

```diff
@@ -72,11 +108,181 @@ fun AssociationProfileScaffold(
+              .width(366.dp)
+              .height(40.dp)
+              .background(Color.LightGray, RoundedCornerShape(12.dp))
+              .padding(vertical = 2.dp, horizontal = 3.dp)
+              .clickable {
+                Toast.makeText(context, "<DEBUG> Not implemented yet", Toast.LENGTH_SHORT).show()
+              },
+  ) {
+    Row(
+        horizontalArrangement = Arrangement.spacedBy(115.dp, Alignment.Start),
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+      Icon(
+          // TODO: Replace with user's profile picture
+          Icons.Filled.Person,
+          contentDescription = "user's profile picture",
+          Modifier.size(36.dp))
+      Text(text = "Casey Rue", style = AppTypography.headlineSmall)
+    }
+  }
+}
+
+@Composable
+fun AssociationProfileEvents(context: Context) {
+  Column(
+      modifier = Modifier.padding(horizontal = 28.dp),
+      horizontalAlignment = Alignment.CenterHorizontally) {
+        Box(modifier = Modifier.testTag("AssociationEventCard")) {
+          EventCard(
+              event =
+                  Event(
+                      organisers = MockReferenceList(),
+                      taggedAssociations = MockReferenceList())) {}
+        }
+        Spacer(modifier = Modifier.size(11.dp))
+        OutlinedButton(
+            onClick = {
+              Toast.makeText(context, "<DEBUG> Not implemented yet", Toast.LENGTH_SHORT).show()
+            },
+            modifier = Modifier.padding(horizontal = 28.dp).testTag("AssociationSeeMoreButton")) {
+              Icon(Icons.AutoMirrored.Filled.ArrowForward, contentDescription = "See more")
+              Spacer(Modifier.width(2.dp))
+              Text(getString(R.string.association_see_more))
+            }
       }
 }
+
+@Composable
+fun AssociationEventTitle() {
+  Text(
+      getString(R.string.association_upcoming_events),
+      modifier = Modifier.padding(horizontal = 20.dp).testTag("AssociationEventTitle"),
+      style = AppTypography.headlineMedium)
+}
+
+@Composable
+fun AssociationDescription() {
+  Text(
+      getString(R.string.debug_lorem_ipsum),
+      style = AppTypography.bodyMedium,
+      modifier = Modifier.padding(horizontal = 24.dp).testTag("AssociationDescription"))
+}
+
+@Composable
+fun AssociationHeader(context: Context) {
+  Row {
+    Box(modifier = Modifier.padding(horizontal = 24.dp).testTag("AssociationImageHeader")) {
+      Image(
+          painter = painterResource(id = R.drawable.adec),
+          contentDescription = "placeholder",
+          modifier = Modifier.size(124.dp))
+    }
+    Column {
+      Text(
+          "xxx followers",
+          style = AppTypography.headlineSmall,
+          modifier = Modifier.padding(bottom = 5.dp).testTag("AssociationHeaderFollowers"))
+      Text(
+          "yyy members",
+          style = AppTypography.headlineSmall,
+          modifier = Modifier.padding(bottom = 14.dp).testTag("AssociationHeaderMembers"))
+      Button(
```

Comment:
Button isn't the same color as in the figma, is it intentional ?

Reply:
The theme of the app defined like this, but I can check if I can do anything to change this